src/frontend/apps/drive/src/features/layouts/components/header/Header.tsx

```diff
@@ -45,40 +46,47 @@ export const HeaderRight = ({
   const [isOpen, setIsOpen] = useState(false);
   const { t } = useTranslation();
   const { config } = useConfig();
+  const authButtonsCustomization = useThemeCustomizationElementsDisplay('auth_buttons');
```

Comment:
We want to stick using the default generic way like `useThemeCustomization("auth_buttons");`